internal/search/query/mapper.go

```diff
@@ -37,6 +40,11 @@ func (*BaseMapper) MapParameter(visitor Mapper, field, value string, negated boo
 	return Parameter{Field: field, Value: value, Negated: negated}
 }
 
+// Base mapper for Patterns. It is the identity function.
+func (*BaseMapper) MapPattern(visitor Mapper, value string, negated bool, quoted bool) Node {
```

Comment:
According to linter:
```suggestion
func (*BaseMapper) MapPattern(visitor Mapper, value string, negated, quoted bool) Node {
```